File: 2089-find-target-indices-after-sorting-array/2089-find-target-indices-after-sorting-array.py
class Solution:
    def targetIndices(self, nums: List[int], target: int) -> List[int]:
        lesser, equal = 0, 0
        for num in nums:
            if num < target:
                lesser += 1
            elif num == target:
                equal += 1
        l = []
        for _ in range(equal):
            l.append(lesser)
            lesser += 1
        return l

File: 2089-find-target-indices-after-sorting-array/2089-find-target-indices-after-sorting-array.cpp
class Solution {
public:
    vector<int> targetIndices(vector<int>& nums, int target) {
        int lesser = 0, equal = 0;
        for (auto num : nums) num < target ? lesser++ : target == num ? equal++ : 0;
        vector<int> res(equal);
        iota(res.begin(), res.end(), lesser);
        return res;
    }
};